config/locales/en.yml

```diff
@@ -103,7 +103,7 @@ en:
       edit:
         info:
         - File taxes online on your own using our partner website. This is our quickest option for households earning under $84,000 to file taxes and collect the tax credits you are owed!
-        - To get started, we’ll need to collect some basic information.
+        - To get started, we'll need to collect some basic information.
```

Comment:
was this a deliberate change? I know we sometimes try to enforce one type of apostrophes vs another.

Reply:
it was not, my editor had a few extensions that really didn't like those. I've reverted for the time being